token/cli/src/main.rs

```diff
@@ -943,6 +943,21 @@ fn command_burn(
     })
 }
 
+
+fn check_mint(
```

Comment:
nit: 
```suggestion
fn check_if_amount_overflows_supply(
```
Or something more descriptive

Reply:
Done